### PR TITLE
Fixes messages not showing after beating a dungeon

### PR DESCRIPTION
**Pull request by NopoTheGamer**
in theory havent tested slight smile

### CODE REVIEW

**Comment by NopoTheGamer** [comment]
have now tested :slightly_smiling_face:

**Comment by hannibal002** [comment]
I'm confused. Your changes should not make any difference at all.

File: src/main/java/io/github/moulberry/notenoughupdates/dungeons/DungeonWin.java [hunk 1]
```diff
@@ -229,12 +229,15 @@ public static void onChatMessage(ClientChatReceivedEvent e) {
 							"Total Damage")
 							|| unformatted.contains("Ally Healing") || unformatted.contains("Enemies Killed") || unformatted.contains(
 							"Deaths") || unformatted.contains("Secrets Found")) {
+							e.setCanceled(true);
 							text.add(e.message.getFormattedText().substring(6).trim());
-						} else {
+						} else if (unformatted.trim().length() > 6) {
 							System.out.println(
 								"These messages would of showed on neu dungeon overlay but didnt, They are either bugged or i missed them: \"" +
```
Comment (hannibal002) [inline src/main/java/io/github/moulberry/notenoughupdates/dungeons/DungeonWin.java:236]: `[..] would of showed [..]` 
:trollface:

**Comment by hannibal002** [comment]
What messages are not shown after a dungeon?

**Comment by NopoTheGamer** [comment]
> What messages are not shown after a dungeon?

--------
dungeon stat one
dungeon stat two
party> important message in party chat 
dungeon stat three
--------
in this example the party chat message would get eaten

**Review by romangraef — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]
Ist das jemals jemanden passiert?

**Pull request merged**